index.d.ts

```diff
@@ -1,18 +1,22 @@
-declare module "react-native-twilio-video-webrtc"{
+declare module "react-native-twilio-video-webrtc" {
   import { ViewProps } from "react-native";
   import React from "react";
 
-  interface TwilioVideoLocalViewProps extends ViewProps {
-    enabled: boolean;
-    ref?: React.Ref<any>;
```

Comment:
Was the `ref` removed intentionally?

Reply:
Nope my bad I moved the interfaces around.
It's there line 15/13 https://github.com/blackuy/react-native-twilio-video-webrtc/pull/407/files#diff-7aa4473ede4abd9ec099e87fec67fd57afafaf39e05d493ab4533acc38547eb8R13

Reply:
Sorry to get back to this, but `TwilioVideoLocalViewProps` is missing the `ref`. The `TwilioVideoParticipantViewProps` still have it, though.

Reply:
That's my bad again, fixed.
Thanks for flagging this